recipe/meta.yaml

```diff
@@ -7,12 +7,12 @@ package:
 
 source:
   fn: {{ name }}-{{ version }}.tar.gz
-  url: https://downloads.kwant-project.org/kwant/kwant-{{ version }}.tar.gz
-  sha256: 8c0ccf341dfa74e1d69f1508968c4d4e9fb36f74685f101897df6a84ed7043df
+  url: https://antonakhmerov.org/misc/kwant-1.4.1.dev20+g40838439.tar.gz
```

Comment:
Don't forget to change this back when the release is made

Reply:
Of course. I'll switch this to PyPi for a source.